jsonify_stored/models/jsonify_mixin.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class JsonifyStored(models.AbstractModel):
+    """Stores the data to export. When inheriting this class,
+       _export_xmlid should be overwritten by an existing export xml_id.
+       Note that this xml_id has to exist before the model init,
+       i.e. either be in a dependency module, or created in a pre_init_hook
+       (however it's fine to modify an existing one in the module data).
+    """
+
+    _name = "jsonify.mixin"
+    _description = "Stores json data"
+
+    _export_xmlid = ""
+
+    data = fields.Serialized(compute="_compute_data", readonly=True, store=False)
+    data_compute = fields.Boolean(
+        compute="_compute_data_compute", default=True, readonly=True, store=True
+    )
+    data_stored = fields.Serialized(
+        compute="_compute_data_stored", readonly=True, store=True
+    )
+
+    @api.model
+    def _get_export(self):
+        xml_id = self._export_xmlid
+        return self.env.ref(xml_id) if xml_id else self.env["ir.exports"]
+
+    @api.model
+    def get_export_depends(self):
+        export_field_names = self._get_export().export_fields.mapped("name")
+        return [fn.replace("/", ".") for fn in export_field_names]
+
+    @api.depends(lambda self: self.get_export_depends())
+    def _compute_data_compute(self):
+        for record in self:
+            record.data_compute = True
+
+    def _compute_data_stored(self):
+        if self:
+            parser = self._get_export().get_json_parser()
+            datas = self.jsonify(parser)
+            for record, data in zip(self, datas):
+                record.data = data
```

Comment:
`data` is not countable
```suggestion
            records_data = self.jsonify(parser)
            for record, data in zip(self, records_data):
                record.data = data
```